cmd/cli/kubectl-kyverno/log/log.go

```diff
@@ -18,7 +18,7 @@ func Configure() error {
 func configure(args ...string) error {
 	logging.InitFlags(nil)
 	if isVerbose(args...) {
-		return logging.Setup(logging.TextFormat, 0)
+		return logging.Setup(logging.TextFormat, logging.DefaultTime, 4)
```

Comment:
why `4` here ?

Reply:
@mviswanathsai is the `4` a mistake ?

Reply:
I might've misunderstood but since it was for `ifVerbose`, I felt that a higher logging level might be suitable

Reply:
Shouldn't the test be `if !isVerbose(args...)` ?